src/sidebar/store/modules/filters.ts

```diff
@@ -118,19 +118,21 @@ function focusFiltersFromConfig(focusConfig: FocusConfig): Filters {
 
 const reducers = {
   CHANGE_FOCUS_MODE_USER(state: State, action: { user: FocusUserInfo }) {
```

Comment:
I would have been clearer if `user: null` was used on the action to represent "no user selected".  I might look at that after this lands.